process/working-group-lifecycle.md

```diff
@@ -31,7 +31,7 @@ Once the WG has further defined its goals and garnered enough support it can app
 * Have a charter or mission statement for review by TAC
 * Have met at least 5 times
     * For these, meeting notes (or ideally recordings) must be public
-* Have at least 5 interested individuals from at least 3 different organizations attending regularly
+* Have at least 5 members from at least 3 different organizations attending regularly
```

Comment:
We've talked before about using "members" to mean people working at companies who financially support the OpenSSF and "contributors" being anyone who is active in a Technical Initiative.

Here (and elsewhere in these changes) do we want to use contributors instead of members? Or do we really mean members here? I think contributors is probably what we want.

Reply:
You're absolutely right. Contributor is a better name for this.
Done.